OneSignalSDK/onesignal/src/main/java/com/onesignal/UserStateSynchronizer.java

```diff
@@ -390,15 +417,15 @@ void onSuccess(String response) {
                         OneSignal.onesignalLog(OneSignal.LOG_LEVEL.DEBUG, "doCreateOrNewSession:response: " + response);
                         JSONObject jsonResponse = new JSONObject(response);
 
-                        if (jsonResponse.has("id")) {
-                            String newUserId = jsonResponse.optString("id");
+                        if (jsonResponse.has(ID)) {
+                            String newUserId = jsonResponse.optString(ID);
                             updateIdDependents(newUserId);
                             OneSignal.Log(OneSignal.LOG_LEVEL.INFO, "Device registered, UserId = " + newUserId);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.